NGitLab/Models/Project.cs

```diff
@@ -224,4 +224,7 @@ public class Project
 
     [JsonPropertyName("mr_default_target_self")]
     public bool? MrDefaultTargetSelf { get; set; }
+
+    [JsonPropertyName("ci_default_git_depth")]
+    public bool? CiDefaultGitDepth { get; set; }
```

Comment:
This property should be an `int?` instead. Refer to [the GitLab doc](https://docs.gitlab.com/api/projects/#get-a-single-project).